Makefile

```diff
@@ -1,8 +1,8 @@
 PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
 PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
-COMMIT_HASH := $(shell git rev-parse --short HEAD)
+VERSION := $(shell git describe --tags --long | sed 's/v\(.*\)/\1/')
```

Comment:
Why do we want to get rid of the `v` prefix?

Reply:
Current `version` command does not print it:
```
alessio@bizet:~/.../github.com/cosmos/cosmos-sdk$ gaiacli version
0.24.2-c6a3928d
```

Reply:
ah OK, that's fine then